{"url": "https://towardsdatascience.com/do-not-use-print-for-debugging-in-python-anymore-6767b6f1866d", "time": 1683019047.671224, "path": "towardsdatascience.com/do-not-use-print-for-debugging-in-python-anymore-6767b6f1866d/", "scrape_status": {"code": 4, "message": "invalid response status code", "resp_status_code": 429}}